drop-boxes/register-omero-metadata/image_registration_process.py

```diff
@@ -89,43 +72,115 @@ def requestOmeroDatasetId(self, project_code=None, sample_code=None):
             sample_code = self._sample_code
 
         cmd_list = list(self._init_cmd_list)
-        cmd_list.append( "python backendinterface.py -p " + str(project_code) + " -s " + str(sample_code) )
+        cmd_list.append( "omero-bifrost query dataset-id " + str(project_code) + " " + str(sample_code) + " --to-xml" )
 
-        commands = ""
-        for cmd in cmd_list:
-            commands = commands + cmd + "\n"
+        out, err = self.executeCommandList(cmd_list)
 
-        process = Popen( "/bin/bash", shell=False, universal_newlines=True, stdin=PIPE, stdout=PIPE, stderr=PIPE )
-        out, err = process.communicate( commands )
-
-        ds_id = str(out)
+        ds_id = -1
+        output_xml = ET.fromstring(str(out))
+        for output_item in output_xml:
+            # print "---> " + str(output_item.tag) + " - " + str(output_item.attrib)
+            ds_id = int(output_item.attrib["id"])
+            break
 
         return ds_id
 
     def registerImageFileInOmero(self, file_path, dataset_id):
 
         cmd_list = list(self._init_cmd_list)
-        cmd_list.append( "python backendinterface.py -f " + file_path + " -d " + str(dataset_id) )
+        cmd_list.append( "omero-bifrost push img-file " + file_path + " " + str(dataset_id) + " --to-xml" )
 
-        commands = ""
-        for cmd in cmd_list:
-            commands = commands + cmd + "\n"
+        out, err = self.executeCommandList(cmd_list)
 
-        process = Popen( "/bin/bash", shell=False, universal_newlines=True, stdin=PIPE, stdout=PIPE, stderr=PIPE )
-        out, err = process.communicate( commands )
+        id_list = []
+        output_xml = ET.fromstring(str(out))
```

Comment:
Very NIT: Since your're importing a package not a constant i'd expect the import name to be lowercase
```suggestion
        output_xml = elementTree.fromstring(str(out))
```